lib/matplotlib/axes/_axes.py

```diff
@@ -2771,29 +2777,19 @@ def errorbar(self, x, y, yerr=None, xerr=None,
           errorbar(x, y, yerr, marker='s',
                    mfc='red', mec='green', ms=20, mew=4)
 
-        where *mfc*, *mec*, *ms* and *mew* are aliases for the longer
-        property names, *markerfacecolor*, *markeredgecolor*, *markersize*
-        and *markeredgewidth*.
+        where mfc, mec, ms and mew are aliases for the longer
+        property names, markerfacecolor, markeredgecolor, markersize
+        and markeredgewidth.
 
         valid kwargs for the marker properties are
 
         %(Line2D)s
 
-        Returns (*plotline*, *caplines*, *barlinecols*):
-
-            *plotline*: :class:`~matplotlib.lines.Line2D` instance
-                *x*, *y* plot markers and/or line
-
-            *caplines*: list of error bar cap
-                :class:`~matplotlib.lines.Line2D` instances
-            *barlinecols*: list of
-                :class:`~matplotlib.collections.LineCollection` instances for
-                the horizontal and vertical error ranges.
-
-        **Example:**
 
+        Example
+        -------
         .. plot:: mpl_examples/statistics/errorbar_demo.py
-
+        
```

Comment:
And there is whitespace as well on this line!

I hope I got all of the whitespace issues.